packages/ActionBar/stories/ShowcaseApp/ShowcaseApp.js

```diff
@@ -89,6 +95,7 @@ export default function App() {
               type={filter.type}
               rule={filter.rule}
               value={filter.value}
+              data={filter.data}
```

Comment:
Since this prop is optional, we can make it clear in this example, `data={filter.columnId === "position" ? data : null}`